eks-public-cluster.tf

```diff
@@ -56,12 +58,13 @@ module "eks-public" {
       instance_types       = ["t4g.xlarge"]
       capacity_type        = "ON_DEMAND"
       min_size             = 1
-      max_size             = 2 # Allow manual scaling when running operations or upgrades
+      max_size             = local.public_cluster_max_size # Allow manual scaling when running operations or upgrades
```

Comment:
nitpick: not sure of the value of moving it to a local (as it's the only ocurence)

Reply:
Moved here as it's a value we'll probably want to change.

Set to 2 as the vpc-cni didn't had enough resources with only one node to be installed.

Reply:
As I understood, it variables to be modified should be in locals.tf or variables.tf so you know what are the actionables.